packages/p2p/src/components/order-details/order-details-confirm-modal.jsx

```diff
@@ -71,7 +71,7 @@ const OrderDetailsConfirmModal = ({
             renderTitle={() => (
                 <Text color='prominent' line-height='m' size='s' weight='bold'>
                     {is_buy_order_for_user ? (
-                        <Localize i18n_default_text='Confirm payment?' />
+                        <Localize i18n_default_text='Payment confirmation' />
                     ) : (
                         <Localize i18n_default_text='Have you received payment?' />
```

Comment:
Could this be refactored into this instead?
```
<Localize in18n_default_text={is_buy_order_for_user ? 'Payment confirmation' : ''Have you received payment?'} />
```

Reply:
Yes. I will refactor that